tests/test_chains.py

```diff
@@ -540,7 +540,44 @@ class Attr(Attr):pass
                              ['Attr -> (Attr -> (Attr -> ()))',
                               'Visitor -> ()'])
             self.assertEqual(c.dump_chains(node.body[-1]), ['Attr -> ()'])
-
+    
+    @skipIf(sys.version_info < (3, 0), 'Python 3 syntax')
+    def test_star_assignment(self):
+        code = '''
+curr, *parts = [1,2,3]
+while curr:
+    print(curr)
+    if parts:
+        curr, *parts = parts
+    else:
+        break
+'''
+        self.checkChains(code, ['curr -> (curr -> (), curr -> (Call -> ()))', 
+                                'parts -> (Starred -> (), parts -> (), parts -> ())']*2)
```

Comment:
I'm a bit puzzled by the `Starred` here.

Reply:
Why is the Starred in the def-use chain? It's not a user, right?

Reply:
OK, I dive a bit and I now understand the situation a bit better. When we use  `*` to expand a container as part of an argument call, the `Starred` is clearly a user. And it' has a `Load` context. When it's used to unpack a tuple, it has a `Store` context and it's just used to specify the nature of the operation. It is debatable whether that's a use or not... I'd rather see that as a property of the def, but we don't have that concept. I don't see it as a new Def either. Let me submit a new PR for that particular behavior